zubhub_frontend/zubhub/package.json

```diff
@@ -23,10 +23,10 @@
     "react": "^17.0.1",
     "react-dom": "^17.0.1",
     "react-i18next": "^11.8.5",
-    "react-quill": "^1.3.5",
+    "react-quill": "^0.0.2",
     "react-redux": "^7.2.2",
     "react-router-dom": "^5.2.0",
-    "react-scripts": "4.0.3",
+    "react-scripts": "^5.0.0",
```

Comment:
it also looks like you upgraded the react script version here. This is not related to the changes you are working on. Like you said, if this was imposed on you by your npm installation, perhaps you should upgrade (or downgrade) your npm installation?